packages/veritone-react-common/src/components/SelectionButton/story.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import SelectionButton from './';
+
+class StatefulSelectionButton extends React.Component {
+  state = {
+    selected: false
+  };
+
+  toggleSelection = evt => {
+    this.setState((prevState, props) => ({
+      selected: !prevState.selected
+    }));
+  };
+
+  render() {
+    return (
+      <SelectionButton
+        selected={this.state.selected}
+        toggleSelection={this.toggleSelection}
+        {...this.props}
+      />
+    );
+  }
+}
+
+class MultipleStatefulSelectionButton extends React.Component {
+  state = {
+    selected: false
+  };
+
+  getSelectionHandler = id => evt => {
```

Comment:
Not sure how I get notification to review this code but this line seems to violate Law of Demeter :trollface: